### DISCUSSION TITLE
Why implicit whitespace doesn't not separate rules?

### DISCUSSION THREAD

**Discussion by Shatur** [pest-parser/pest#946] [Q&A] [answered]
I have the [following grammar](https://pest.rs/?g=N4Ig5gTghgtjURALhAMwK4DsDGAXAlgPaYAEAvCQPrAkDKA8gJIkB%2BJApgB4AOErJAUSYkAvgB1MXXuSo0Abgn4AKfJlT5OAKn4KIASm3jMq9Zxk0xITZdETd5kgEFaAYUaNKjgDIAFABKOANS2mBIA6n6MACoCtD6OLgIy1CQ2NiIgADQgqtzouMggjiQAQiAiQA#editor).
With input `A B` it captures `A B` as a single `var`. But why? I would expect it to capture `A` and `B` as separate vars. In this case I trying to make multiplication implicit and want to skip all spaces. I can insert my own whitespace rule manually, but I would expect implicit whitespace rule to behave the same.

**Comment by NyalephTheCat** [comment, marked as answer]
It's probably because var doesn't have an @ at the start to say that is it a rule without whitespace 

You should have `var = @{ ASCII_ALPHA+ }`

**Reply by Shatur** [reply]
Oh, right, thanks!
